### PR TITLE
create-api-key.md updated

### PR DESCRIPTION
**Pull request by jesusgonzalez12**
This version introduces improvements on how a user can work with creating API keys using the pyc8 SDK.
Only changes to the pyc8 code have been introduced.
You can check for more details on the Kanban board.

### CODE REVIEW

File: docs/sdks/create-api-key.md [hunk 1]
```diff
@@ -29,8 +29,23 @@ client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
...
+# Check wether the chosen_api_key exists or not, comparing it to the list elements
+for apiElement in apiKeyIds:
+    if apiElement == CHOSEN_API_KEY_ID:
+        API_ACTIVE = True
+        break
+
+# Checks if the API key already exists and acts accordingly
+if API_ACTIVE:
+    # Send message that the key already exists
+    print("Remove API Key: ", client.remove_api_key(CHOSEN_API_KEY_ID))
```
Comment (jesusgonzalez12) [inline docs/sdks/create-api-key.md]:
```suggestion
    print("Can't create an already existing key")
```

File: docs/sdks/create-api-key.md [hunk 2]
```diff
@@ -29,8 +29,23 @@ client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                     email='nemo@nautilus.com', password='xxxxx',
                     geofabric='_system')
 
-# Create an API key
-print("Create API Key: ", client.create_api_key('id1'))
+# Check wether the chosen_api_key exists or not, comparing it to the list elements
```
Comment (jamesm-macrometa) [inline docs/sdks/create-api-key.md]:
```suggestion
# Checks if `chosen_api_key` exists
```

Comment (jamesm-macrometa) [reply docs/sdks/create-api-key.md]: Same comment as on other PR, not sure what we mean by "list elements" but we might want to add that part back in.

File: docs/sdks/create-api-key.md [hunk 3]
```diff
@@ -29,8 +29,23 @@ client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
...
+for apiElement in apiKeyIds:
+    if apiElement == CHOSEN_API_KEY_ID:
+        API_ACTIVE = True
+        break
+
+# Checks if the API key already exists and acts accordingly
```
Comment (jamesm-macrometa) [inline docs/sdks/create-api-key.md]:
```suggestion
# Checks if the API key exists
```

File: docs/sdks/create-api-key.md [hunk 4]
```diff
@@ -29,8 +29,23 @@ client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
...
+if API_ACTIVE:
+    # Send message that the key already exists
```
Comment (jamesm-macrometa) [inline docs/sdks/create-api-key.md]:
```suggestion
    # Display a message that the key already exists
```

**Review by jamesm-macrometa — CHANGES_REQUESTED** [review]
A few minor changes, looks good!

File: docs/sdks/create-api-key.md (hunk 2 above)
Comment (jesusgonzalez12) [reply docs/sdks/create-api-key.md]: Sorry, I missed adding a few lines of code. Fixed.

A list of all API key IDs a user has is saved in a variable which is used to make a comparison check later.

**Comment by jesusgonzalez12** [comment]
Thanks for the review.
Please, check again the element list topic.

File: docs/sdks/create-api-key.md (hunk 2 above)
Comment (jamesm-macrometa) [reply docs/sdks/create-api-key.md]: Got it. I'm not certain it's necessary to mention that much detail in this particular message.  @diana-macrometa any thoughts?

Comment (jesusgonzalez12) [reply docs/sdks/create-api-key.md]: I have updated the comment to improve semmantics.

File: docs/sdks/create-api-key.md [hunk 5]
```diff
@@ -29,8 +29,28 @@ client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
...
                     geofabric='_system')
 
-# Create an API key
-print("Create API Key: ", client.create_api_key('id1'))
+# Get a list of all the API key ID's names
+apiKeyIds = []
+for api in client.list_all_api_keys():
+    apiKeyIds.append(api["keyid"])
+
+# Check wether the chosen_api_key exists or not, comparing it to the API key list
```
Comment (jamesm-macrometa) [inline docs/sdks/create-api-key.md]:
```suggestion
# Checks if `chosen_api_key` exists
```

**Review by jamesm-macrometa — APPROVED** [review]

**Pull request merged**